physics/boyles_law.py

```diff
@@ -0,0 +1,181 @@
+"""
+Title : Implementation of Boyle's law.
+
+Description :
+    Boyle's law, also referred to as the Boyle-Mariotte law, or Mariotte's law
+    (especially in France), is an experimental gas law that describes the relationship
+    between pressure and volume of a confined gas.
+
+    Boyle's law is a gas law which states that the pressure exerted by a gas
+    (of a given mass, kept at a constant temperature) is inversely proportional to the
+    volume occupied by it.
+
+    In other words, the pressure and volume of a gas are inversely proportional to each
+    other as long as the temperature and the quantity of gas are kept constant.
+    Boyle's law was put forward by the Anglo-Irish chemist Robert Boyle in the year 1662
+
```

Comment:
```suggestion
```
I feel that this part is redundant and/or unneeded info

Reply:
Ok, will remove this part. Thanks !!